certbot/certbot/util.py

```diff
@@ -567,6 +560,21 @@ def enforce_domain_sanity(domain):
     return domain
 
 
+def is_ipaddress(address):
+    """this function check if input name is actually an IP(v4 or v6) address"""
+    try:
+        socket.inet_pton(socket.AF_INET, address)
+        # If this line runs it was ip address (ipv4)
+        return True
+    except socket.error:
+        # It wasn't an IPv4 address, so try ipv6
+        try:
+            socket.inet_pton(socket.AF_INET6, address)
```

Comment:
This will cause a warning about statement has no effect. Writing `_ = ....` or adding `#  noqa` at EOL will remove that warning. It's nothing serious. I think there are already loads of things like this in the code base.